src/components/introduction.stories.mdx

```diff
@@ -4,13 +4,13 @@
 
 React components for building realtime meeting applications with Amazon Chime.
 
-- [UI Components](/?path=/docs/ui-components-badge--basic-badge)
-- [SDK Components](/?path=/docs/sdk-components-introduction--page)
-- [UI Providers](/?path=/docs/ui-providers-notificationprovider--page)
-- [SDK Providers](/?path=/docs/sdk-providers-introduction--page)
-- [UI Hooks](/?path=/docs/ui-hooks-notification-usenotificationdispatch--page)
-- [SDK Hooks](/?path=/docs/ui-hooks-notification-usenotificationdispatch--page)
-- [Themes](/?path=/docs/themes--page)
+- [UI Components](/story/ui-components-badge--basic-badge)
+- [SDK Components](/story/sdk-components-introduction--page)
+- [UI Providers](/story/ui-providers-notificationprovider--page)
+- [SDK Providers](/story/sdk-providers-introduction--page)
+- [UI Hooks](/story/ui-hooks-notification-usenotificationdispatch--page)
+- [SDK Hooks](/story/ui-hooks-notification-usenotificationdispatch--page)
+- [Themes](/story/themes--page)
```

Comment:
I tried to replace the suffix with these new ones in current storybook, and it leads me to a 404 page. Could you give me a full URL example?

Reply:
It seems we need to keep the "/?path=" part.
```
/?path=/docs/themes--page // Before
/?path=/story/themes--page // After
```

Reply:
Oh, I get it, the "/?path=" will be added automatically.

Reply:
It will automatically add the `/?path=` for us. The current links would navigate to something like `/?path=/?path=/docs/`.

```
https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/?path=/docs/themes--page // current link
https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/themes--page // expected link